src/js/footer/featureFlagLiveChat/index.js

```diff
@@ -0,0 +1,24 @@
+// Import Firebase SDK modules
+import { initializeApp } from "https://www.gstatic.com/firebasejs/10.8.1/firebase-app.js";
```

Comment:
why not we send a fetch request instead of using this?
basically you need to fetch this endpoint directly:
`https://app-config-prod.firebaseio.com/remote_config/deriv-com/chat`

Reply:
we can fetch yes but , it should work in real time i think
once they turn off we need show the change instantly  so that why we need a firebase ws  connection

Reply:
that's right. ok lets do it this way then